ls1intum/artemis-android
# `Bugfix`: Fix adding members to conversation

### Problem Description

There was an issue that prevented users from adding members to a conversation. Instead of adding the members, an error was shown.

### Changes

The cause of this error was a missing conversation id that ensures a match to the correct conversation. The manage_conversations_module has been changed to correctly set the conversationId parameter of the ConversationAddMembersViewModel. This PR fixes #139.
In Conversation AddMemberSettingsE2eTest, the functionality of adding members is tested. However, this bug did not affect the tests as the conversationId is set independently from the module there.

### Steps for testing

- Go to a group
- Click the settings icon
- Click `Add members`
- Select members, click `Add`
- Verify that the members get added correctly
